pilot/cmd/pilot-agent/main.go

```diff
@@ -88,6 +88,8 @@ var (
 	tlsClientRootCert          string
 	tlsCertsToWatch            []string
 	loggingOptions             = log.DefaultOptions()
+	sdsEnabled                 bool
```

Comment:
This should be based on checking if the file exists, it can't be a global setting ( upgrade, etc)